modules/seqexec/web/shared/src/main/scala/seqexec/web/model/boopickle/ModelBooPicklers.scala

```diff
@@ -236,6 +195,47 @@ trait ModelBooPicklers extends GemModelBooPicklers {
   implicit val sequenceQueueViewPickler =
     generatePickler[SequencesQueue[SequenceView]]
 
+  private val comaOptionIdx = Map((0 -> ComaOption.ComaOn),
+                                  (1 -> ComaOption.ComaOff))
+
+  implicit val comaPickler = valuesMapPickler(comaOptionIdx)
+
+  private val tipTiltSourceIdx = Map((0 -> TipTiltSource.PWFS1),
+                                     (1 -> TipTiltSource.PWFS2),
+                                     (2 -> TipTiltSource.OIWFS),
+                                     (3 -> TipTiltSource.GAOS))
```

Comment:
I wonder if this too could be generalized for any `Enumerated` instance.

Reply:
Right, that should be possible too